docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml

```diff
@@ -338,6 +338,13 @@
           Those credentials are then used to authenticate any requests from
           those tools.</para>
         </sect3>
+
+        <sect3 id="TopoSvr">
+          <title>Topology Server</title>
+
+          <para>The topology server is an internal component used by ROXIE.
```

Comment:
...by ROXIE to keep track of the health of the different roxie processes in a cluster.

We should at least have a minimal comment about what it does.

Reply:
It was a one-off specific to this file - we use only the one XML editor for any files destined for the repo just for that reason.